base_miner/UCF/config/__init__.py

```diff
@@ -0,0 +1,7 @@
+import os
+import sys
+current_file_path = os.path.abspath(__file__)
+parent_dir = os.path.dirname(os.path.dirname(current_file_path))
+project_root_dir = os.path.dirname(parent_dir)
+sys.path.append(parent_dir)
+sys.path.append(project_root_dir)
```

Comment:
[Question] is this necessary when the project is installed in editable mode with `pip install -e .` ?

Reply:
Will remove these type of imports and refactor where necessary in a subsequent small PR. Thanks!